bindings/rust-examples/hyper-server-client/README.md

```diff
@@ -0,0 +1,15 @@
+This example demonstrates how to use s2n-tls with the [hyper](https://hyper.rs/) HTTP library.
+
+The server example demonstrates how to use s2n-tls with the [hyper-util server](https://docs.rs/hyper-util/latest/hyper_util/server/conn/auto/struct.Builder.html). The client example demonstrates how to use s2n-tls with the [legacy hyper-util client](https://docs.rs/hyper-util/latest/hyper_util/client/legacy/struct.Builder.html), via the [s2n-tls-hyper](../../rust/standard/s2n-tls-hyper) compatibility crate.
```

Comment:
Should we explain why we only support the legacy client?

Reply:
Hmm, well, there isn't currently a non-legacy hyper-util client. I think it's just considered legacy because it was moved from hyper 0.x into hyper-util during the hyper1 migration. But this detail probably isn't too relevant to the example, so I might just remove the term legacy from this description, if that seems reasonable.